test/Microsoft.Health.Dicom.Web.Tests.E2E/Rest/ExportTests.cs

```diff
@@ -93,102 +93,90 @@ public async Task GivenFiles_WhenExporting_ThenSuccessfullyCopy()
             { DicomIdentifier.ForInstance(instance9), instance9 },
         };
 
-        BlobContainerClient containerClient = CreateContainerClient();
-
         // Upload files
         await Task.WhenAll(instances.Select(x => _instanceManager.StoreAsync(new DicomFile(x.Value))));
 
-        // Create new container if necessary
-        await containerClient.CreateIfNotExistsAsync(PublicAccessType.None);
-
-        try
-        {
-            // Begin Export
-            DicomWebResponse<DicomOperationReference> response = await _client.StartExportAsync(
-                ExportSource.ForIdentifiers(
-                    DicomIdentifier.ForStudy(studyUid1),
-                    DicomIdentifier.ForSeries(studyUid2, seriesUid2),
-                    DicomIdentifier.ForInstance(instance7),
-                    DicomIdentifier.ForInstance(unknownStudyUid1, unknownSeriesUid1, unknownSopInstanceUid1),
-                    DicomIdentifier.ForSeries(unknownStudyUid2, unknownSeriesUid2),
-                    DicomIdentifier.ForInstance(instance8),
-                    DicomIdentifier.ForInstance(instance9),
-                    DicomIdentifier.ForStudy(unknownStudyUid3)),
-                _options.Destination);
-
-            // Wait for the operation to complete
-            DicomOperationReference operation = await response.GetValueAsync();
-            IOperationState<DicomOperation> state = await _client.WaitForCompletionAsync(operation.Id);
+        // Begin Export
```

Comment:
This looks like a complex change in the diff, but I just removed the `try`/`catch` block